nima_core/precognition/engine.py

```
@@ -47,6 +48,45 @@
+        for pat in patterns:
+            key = _pattern_key(pattern_type, pat)
+            if not key:
+                continue
+            storage.store_pattern(
+                pattern_type=pattern_type,
+                pattern_key=key,
+                pattern_json=json.dumps(pat),
+                confidence=pat.get("confidence", 0.0),
+                evidence_count=pat.get("observation_count", pat.get("count", 0)),
+            )
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)